src/lib/Ska/Starcheck/FigureOfMerit.pm

```diff
@@ -28,8 +28,8 @@ use Inline Python => q{
 
 from chandra_aca.star_probs import acq_success_prob, prob_n_acq, mag_for_p_acq
 
-def _acq_success_prob(date, t_ccd, mag, color, spoiler):
-    out = acq_success_prob(date, float(t_ccd), float(mag), float(color), spoiler)
+def _acq_success_prob(date, t_ccd, mag, color, spoiler, halfwidth):
+    out = acq_success_prob(date, float(t_ccd), float(mag), float(color), spoiler, int(halfwidth))
```

Comment:
But apparently not enough to check this and add a helpful comment to the actual code in this PR